### PR TITLE
Create TH-MOI

### PR DESCRIPTION
**Pull request by AudreyIATI**
Related to [Issue 503](https://github.com/org-id/register/issues/503)

### CODE REVIEW

File: lists/th/th-moi.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "licenseDetails": ""
+  },
+  "links": {
+    "opencorporates": "",
+    "wikipedia": "https://en.wikipedia.org/wiki/Ministry_of_Interior_(Thailand)"
+  },
+  "formerPrefixes": [],
+  "meta": {
+    "source": "Original Research",
+    "lastUpdated": "25/03/22"
```
Comment (kd-ods) [inline lists/th/th-moi.json:44]: Please put the date in YYYY-MM-DD format

**Review by kd-ods — CHANGES_REQUESTED** [review]
A few things to address before we can merge this.

**Comment by AudreyIATI** [comment]
Changes made - let me know if you need anything else from me.

Thanks,
Audrey

**Comment by amy-silcock** [comment]
Let me know if there are further items to change on this. As well as what needs to be changed so all checks pass.

**Pull request merged**